site/content/docs/contributing/getting-started.md

```diff
@@ -20,44 +20,53 @@ an account to contribute.
 
 ## 2. Install Tools
 
-### Install Documentation Tools
+### Install git
 
-If you wish to contribute to the documentation, it is recommended but not 
-required to install [hugo], which we use to develop this site.
-
-Please see: https://gohugo.io/getting-started/installing/
-
-### Install Developer Tools
-
-If you wish to contribute to kind's code you will need to install the following:
+Our source code is managed with [`git`][git], to develop locally you
+will need to install `git`.
 
-* `git`
-* `go`
-* `docker`
-
-#### Install git
-Install `git` on your local machine.
 You can check if `git` is already on your system and properly installed with 
 the following command:
 
 ```
 git --version
 ```
-This documentation is written using `git` version 2.17.1. 
+This documentation is written using `git` version `2.17.1`. 
```

Comment:
written -> managed?

also possibly no need to add this sentence?
users without git will land with a fairly new and compatible git package as a download.

Reply:
SGTM, this was already in the doc, I only changed the command / version markup to match. Will remove.

Reply:
done